BUILD.md

```diff
@@ -422,6 +422,26 @@ proxy binary:
 bin/docker-build-proxy
 ```
 
+#### Locally built proxy
+
+If you want to deploy a locally built proxy, you can build it in the [`linkerd2-proxy`](https://github.com/linkerd/linkerd2-proxy) repo by running:
```

Comment:
To fix the markdown lint you'll need to make sure this is wrapped at column 80.

Reply:
Done.